core/cmd/renderer_test.go

```diff
@@ -275,11 +276,15 @@ func TestRendererTable_ServiceAgreementShow(t *testing.T) {
 	buffer := bytes.NewBufferString("")
 	r := cmd.RendererTable{Writer: buffer}
 
-	assert.NoError(t, r.Render(&psa))
+	require.NoError(t, r.Render(&psa))
 	output := buffer.String()
-	assert.Regexp(t, regexp.MustCompile("0x[0-9a-zA-Z]{64}"), output)
-	assert.Regexp(t, regexp.MustCompile("1.000000000000000000 LINK"), output)
-	assert.Regexp(t, regexp.MustCompile("300 seconds"), output)
+	checkPresence := func(s string) { assert.Regexp(t, regexp.MustCompile(s), output) }
+	checkPresence("0x[0-9a-zA-Z]{64}")
+	checkPresence("1.000000000000000000 LINK")
+	checkPresence("300 seconds")
+	checkPresence("0xDeaDbeefdEAdbeefdEadbEEFdeadbeEFdEaDbeeF") // Aggregator address
+	checkPresence("0xd0771e55")                                 // AggInitiateJobSelector
+	checkPresence("0xbadc0de5")                                 // AggFulfillSelector
```

Comment:
I may be wrong, though I believe the use of anonymous functions in this way is not very go'ey.

Reply:
Thanks, @felder-cl . Fixed in bd1ae8f3cb86fdebb95df1468d5495627d8a14f3